packages/calling-component-bindings/src/hooks/usePropsFor.ts

```diff
@@ -179,7 +179,7 @@ const findSelector = (component: (props: any) => JSX.Element | undefined): any =
   return undefined;
 };
 
-/* @conditional-compile-remove(PSTN-calls) */
+/* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(notifications) */
```

Comment:
You should be moving your selector out of this function into the main selector mapping. this is for selectors that need to be cc'd out